src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -311,7 +318,8 @@ AcquireExecutorMultiShardLocks(List *taskList)
 			 * and therefore prevents other modifications from running
 			 * concurrently.
 			 */
-
+			LockParentRelationShardResourcesIfPartition(task->relationShardList,
```

Comment:
I suppose we should never be doing anything here since we don't support RF >1, right?